src/components/Extensibility/components/RadialChart.jsx

```diff
@@ -49,11 +49,11 @@ export const RadialChart = ({ structure, value, originalResource }) => {
         color={structure.color}
         value={value}
         max={maxValue ? maxValue : structure?.maxValue}
-        titleText={t(structure?.name)}
+        titleText={t(structure?.name, structure?.name ?? '')}
```

Comment:
Shouldn't it be done this way?
```suggestion
        titleText={t(structure?.name, {defaultValue: structure?.name ?? ''})}
```